### PR TITLE
Create a new user to run 0MQ subscribers in screens (+rc.local +permissions)

### PR DESCRIPTION
**Pull request by gallypette**
- adds a new user: zmqs
- modifies rc.local to run /var/www/misp-dashboard/start_zmq.sh (launch screens)
- modifies permissions on /var/www/misp-dashboard so members of www-data (zmqs) can create a log folder there.

The default config does not match the changes yet, hence the draft

### CODE REVIEW

File: scripts/users.sh [hunk 1]
```diff
@@ -10,3 +10,6 @@ echo %misp ALL=NOPASSWD:ALL > /etc/sudoers.d/misp
 ##chmod 0440 /etc/sudoers.d/thehive
 chmod 0440 /etc/sudoers.d/misp
 
+echo "--- Creating zmqs user"
+useradd -U -G www-data -m -s /bin/bash  zmqs
+echo zmqs:zmqs"Password1234" | chpasswd
```
Comment (SteveClement) [inline scripts/users.sh]: Does it require a new user with password?
This is potentially scary ;)

Comment (gallypette) [reply scripts/users.sh]: Well yes, I should definitely remove that line :)

**Comment by adulau** [comment]
@gallypette is the PR still a draft? Could you review it when you have time.

**Comment by SteveClement** [comment]
This needs to be included in the misp-dashboard documentation: 

https://github.com/MISP/MISP/blob/2.4/docs/generic/misp-dashboard-debian.md

bootstrap.sh is obsolete.

**Comment by gallypette** [comment]
I overlooked the fact that bootstrap.sh was obsolete.
So in the end nothing should be done in misp-packer, but in https://github.com/MISP/MISP/blob/2.4/docs/generic/misp-dashboard-debian.md ?

**Comment by SteveClement** [comment]
@gallypette exactly :)

**Pull request closed without merging**